data/hai/unfettered side missions.txt

```diff
@@ -0,0 +1,274 @@
+	on fail
+		"reputation: Hai (Unfettered)" <?= -1000
+	on complete
+		dialog `You receive a message from the 'associates of Hai-home', "thank you for helping our brethren, <last>. We will not forget this. Please accept these <payment> to get you started on paying back."`
+		payment 1250000
+
+
+
+outfit "Unfettered Landing License"
+	category "Licenses"
+	thumbnail "outfit/unknown"
+	"maintenance costs" 600
+	description "This is needed in order to land on Unfettered planet, but can be sold at any time in order to cancel the arrangement."
+
+mission "Unfettered Relic of old"
+	landing
+	source "Firelode"
+	to offer
+		has "ship model: Ancient Ladybug"
+	on offer
+		conversation
+			`As you land and get out of your ship you quickly notice you have attracted a lot of attention, probably due to this old design. One Unfettered that looks like an engineer comes to speak to you, seeming much more open to speak than usual. "Welcome. I do not know how you have obtained this ship, and some will probably see it as a sign. As you may know it is a relic of our past, a past we are quite proud of."`
+			`	"In truth, I do not personally care about all of that. All I care about is that this ship is old, and has technology that would be very valuable because of how rare it is. Thus, I challenge you for it!"`
+			choice
+				`	"What do I get if I win?"`
+					goto win
+				`	"Can't I just sell it to you?"`
+			`	"No, because in the eyes of my brethren that would mean I did not deserve this ship."`
+			label win
+			`	"You will win the right to access our worlds for just a meagre daily fee of 600 credits. You will not be allowed to land like today any other time should you refuse this deal. Beware, for there are no rules, other than the fact you may only use one ship, and only one ship! Tell me then, what say you, will you take to the skies, or are you too effrayed?"`
+			choice
+				`	"I accept."`
+				`	"I have nothing to prove to you."`
+					decline
+			`	"Good! I will of course not be fighting you personally, as it is too risky to do so."`
+				accept
+	on decline
+		"reputation: Hai (Unfettered)" >?= -1000
+	on fail
+		"reputation: Hai (Unfettered)" >?= -1000
+		dialog `The Unfettered hail you, "You failed the mission, and as such, are no longer authorized to land on our worlds, and you will not enjoy unmerited respect any longer!"`
+	npc
+		government "Hai (Unfettered Challenger)"
+		to spawn
+			"total ships" == 1
+		to despawn
+			"total ships" != 1
+		personality unconstrained launching plunders staying target heroic
+		ship "Lightning Bug" "Thunder"
+		ship "Lightning Bug (Unfettered Shipyards)" "Struck"
+		ship "Lightning Bug (Tripulse)" ""
```

Comment:
```suggestion
		ship "Lightning Bug (Tripulse)" "Lightning"
```
This should have a name.

Reply:
Yeah I went with bug but didnt push the last version aparently, woops

Reply:
```suggestion
		ship "Lightning Bug (Tripulse)" "Erazer"
```